packages/sdk/src/Sygma.ts

```diff
@@ -148,6 +151,7 @@ export class Sygma implements SygmaSDK {
     }
 
     if (!chain1) {
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
it is unreasonable to use `eslint-disable-next-line` on this one, please look at the issue and solve it in the correct way.